test/spec-runner.html

```diff
@@ -19,10 +19,15 @@
 
 		<!-- JavaScripts -->
 		<script src="/js/navigation-search.js"></script>
-
+		<script src="/js/alert-box.js"></script>
+		<script src="/js/utility/namespacer.js"></script>
+		<script src="/js/alert-box.js"></script>
```

Comment:
Uneeded reference, you referenced this in line 22.

Reply:
Done!